				changed	offset
13	8	13	10	2	0
384	10	386	65	55	2
404	7	461	7	0	57
415	6	472	9	3	57
444	6	504	14	8	60
					68

not sure what is wrong. don't edit the file with textlime, just git diff filea fileb > 1.patch then vi 1.patch

there is 68 offset after previous patches
2143 + 68 = 2211
--- a/src/rfc1035.c
+++ b/src/rfc1035.c
@@ -2211,7 +2211,7 @@ size_t answer_request(struct dns_header *header, char *limit, size_t qlen,
       if (qtype == T_MAILB)
        ans = 1, nxdomain = 1, sec_data = 0;

-      if (qtype == T_SOA && option_bool(OPT_FILTER))
+      if ((qtype == T_SOA || qtype == 65) && option_bool(OPT_FILTER))
        {
          ans = 1;
          sec_data = 0;
          
